docs/user-guide/cli-using-editing-team-configuration.md

```diff
@@ -0,0 +1,43 @@
+# Editing team configurations
+
+After you [initialize team configuration](../user-guide/cli-using-initializing-team-configuration), the newly created team profiles need additional details before they can be shared and applied in your environment. This could include information such as a port number, or user credentials.
```

Comment:
"This could include information such as a port number, or user credentials."
remove comma
"This could include information such as a port number or user credentials."